apps/web/components/shared/project-card-variants.tsx

```
@@ -0,0 +1,384 @@
+                <div>
+                    <div className="text-3xl font-bold">$45,678</div>
+                    <div className="text-gray-500">raised of $100,000</div>
+                </div>
+                <div className="text-right">
+                    <div className="text-3xl font-bold">15</div>
+                    <div className="text-gray-500">days left</div>
+                </div>
+            </div>
+
+            <div className="w-full h-2 bg-gray-200 rounded-full mb-6">
+                <div className="w-5/12 h-2 bg-blue-500 rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)